examples/src/main/java/io/dapr/examples/unittesting/DaprWorkflowExampleTest.java

```diff
@@ -42,7 +42,7 @@ public class DaprWorkflowExampleTest {
   private static final String noTimeoutWorkflow = "DemoWorkflowNoTimeout";
   private static final String workflowDefaultId = "demo-workflow-123";
 
-  private class DemoWorkflow extends Workflow {
+  private class DemoWorkflow implements Workflow {
```

Comment:
Java docs is updated only after a Dapr release. It reflects the latest release SDK, not master. It is used to populate our GitHub Website.

Reply:
Done